### PR TITLE
 GSAGH-458-Add-assemblies-to-GetModelGeometry-component

### PR DESCRIPTION
**Pull request by tlmnrnhrdt**

### CODE REVIEW

File: GsaGH/Components/0_Model/GetModelGeometry.cs [hunk 1]
```diff
@@ -269,7 +270,7 @@ private enum FoldMode {
       Params.Output[i].Name = "3D Members in [" + unitAbbreviation + "]";
 
       i = 1;
-      for (int j = 1; j < 7; j++) {
+      for (int j = 1; j < 8; j++) {
```
Comment (DominikaLos) [inline GsaGH/Components/0_Model/GetModelGeometry.cs:273]: what is 8? can you change this magic number?

File: GsaGH/Helpers/GsaAPI/Mappings.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 namespace GsaGH.Helpers.GsaApi {
   internal static class Mappings {
-    internal static readonly Dictionary<string, AlignmentType> alignmentTypeMapping
+    internal static readonly Dictionary<string, AlignmentType> AlignmentTypeMapping
```
Comment (DominikaLos) [inline GsaGH/Helpers/GsaAPI/Mappings.cs:13]: this should be lower case
image

**Review by DominikaLos — CHANGES_REQUESTED** [review]

File: GsaGH/Components/0_Model/GetModelGeometry.cs (hunk 1 above)
Comment (tlmnrnhrdt) [reply GsaGH/Components/0_Model/GetModelGeometry.cs:273]: that is the number of outputs - 1?!

File: GsaGH/Helpers/GsaAPI/Mappings.cs (hunk 2 above)
Comment (tlmnrnhrdt) [reply GsaGH/Helpers/GsaAPI/Mappings.cs]: changed

**Review by DominikaLos — APPROVED** [review]

**Pull request merged**